guide/inheritance-wallet/wallet-creation.md

```diff
@@ -0,0 +1,338 @@
+
+-->
+
+# Wallet creation 
+{: .no_toc }
+
+---
+
+<div class="glossary-toc" markdown="1">
+ * Table of contents
+{:toc}
+</div>
+
+---
+
+## Onboarding
+
+Now that we have a [good understanding]({{ '/guide/inheritance-wallet/overview/' | relative_url }}) of how Bob and Alice want to set up their inheritance wallet, let's take a close look at the first use experience and wallet setup.
+
+After downloading the app, the app walks them through the purpose of the product and highlights some of the most important features. 
+
+{% include image-gallery.html pages = page.images_wallet-creation %}
+
+## Wallet creation
+
+After going through the app onboarding flow, Bob and Alice are ready to create their savings wallet. The wallet creation flow consists of three main tasks: 
+
+1. Configure the wallet
+2. Import the signing keys
+3. Review and finalize the wallet
+
+Try out the prototype below. It covers the entire wallet creation process.
+
+{% include prototype.html
+   link = "https://www.figma.com/proto/h5GP5v5dYfpXXfEUXf6nvC/Inheritance-wallet?page-id=5542%3A2120&type=design&node-id=6227-6395&viewport=632%2C2194%2C0.09&t=GM6AiUnmOWB7jsQs-1&scaling=min-zoom&starting-point-node-id=6227%3A6395&show-proto-sidebar=1&mode=design"
+   image = "/assets/images/guide/inheritance-wallet/wallet-creation/prototype.jpg"
+   retina = "/assets/images/guide/inheritance-wallet/wallet-creation/prototype@2x.jpg"
+   mobile = "/assets/images/guide/inheritance-wallet/wallet-creation/prototype-mobile.jpg"
+   mobileRetina = "/assets/images/guide/inheritance-wallet/wallet-creation/prototype-mobile@2x.jpg"
+   alt-text = "A couple of screens from the reference design prototype"
+   width = 800
+   height = 500
+%}
+
+## Step 1: Wallet configuration
+
+The first is the wallet configuration, where Bob and Alice define the rules for how the wallet should work. In the second part, they add all the signing keys.
+
+#### Creating the primary key set
+
+Bob starts the wallet creation process on his phone, while Alice will use the app on her own phone to add her signing key. The app gives them a choice between two of the most commonly used wallet types: The first one is a standard, single-key wallet. The second option is a 2-of-3 multi-key wallet, which is suited for larger savings. There is also a third option that allows users to create their own custom setups. Bob and Alice choose the 2-of-3 multi-key wallet.
+
+{% include image-gallery.html pages = page.images_wallet-configuration %}
+
+#### Enabling the recovery path
+
+After configuring the primary key set, the app asks the users whether they would like to enable a recovery path. The recovery path uses the same keys that are part of the primary key set and unlocks after 6 months of inactivity. If it is unlocked, only one signature will be required to spend bitcoin, instead of two. We have covered this in the time-based reference design as well.
```

Comment:
```suggestion
After configuring the primary key set, the app asks the users whether they would like to enable a recovery path. The recovery path uses the same keys that are part of the primary key set and unlocks after 6 months of inactivity. If it is unlocked, only one signature will be required to spend bitcoin, instead of two. We have covered this in the time-based reference design as well.
```
Turn `time-based reference design` into a link to that page.